docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -589,33 +589,7 @@ export default function CssModulesSliderDeep2() {
 
 Emotion's **css()** method works seamlessly with Material-UI.
 
-{{"demo": "pages/guides/interoperability/EmotionCSS.js", "hideToolbar": true}}
-
-[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/emotion-interoperability-idymy)
-
-```jsx
-/** @jsx jsx */
-import { jsx, css } from '@emotion/react';
-import Slider from '@material-ui/core/Slider';
-
-export default function EmotionCSS() {
-  return (
-    <div>
-      <Slider defaultValue={30} />
-      <Slider
-        defaultValue={30}
-        css={css`
-          color: #20b2aa;
-
-          :hover {
-            color: #2e8b57;
-          }
-        `}
-      />
-    </div>
-  );
-}
-```
```

Comment:
I don't understand. Why are we removing the codesandbox and the code preview? These demos are meant to show that it works in isolation. It doesn't matter that it works in our dev environment. We use `"hideToolbar": true` as much as possible.

Reply:
> We have used codesandbox and "hideToolbar": true as much as possible.

Why would we use it as much as possible?

> Why remove the codesandbox and the inlined JSX? 

Because we have existing code that does it for us.

Reply:
> Why would we use it as much as possible?

The idea was to have as much as possible outside of the documentation to be isolated. However, it comes with a maintenance cost. I believe we have both use a live demo (only demonstrated the final output, with hidden sources), as well as showing the source for the advanced portal demo.

Reply:
Oh, I definitely agree with the multi-file demos. But for single file demos that work just fine in our demo setup I don't see why we wouldn't use it.

Reply:
Yeah, agree, especially if the edit to codesandbox button in the toolbar works as-is.